### ISSUE TITLE
Juce serial port linux

### ISSUE THREAD

**Issue by giancarlocp** [cpr2323/juce_serialport#1]
Hello Chris @cpr2323, I was thinking to add libserial to a Juce project to test until I saw this, great job.
I am new in Juce. I will like to test both option, your lib and libserial. Some help/guide to put this working in linux will be appreciated. what is the difference in working with this instead the libserial?

**Comment by cpr2323** [comment]
Hello, sorry for my belated response. The library does not have Linux support at this time. And, actually, it's not really 'my' library, as I picked it up from the JUCE forums quite some time ago, but it was never hosted anywhere others could get to it easily, so I put it on my github, and I have fixed a couple of things with it. What makes it useful, is that it is a JUCE module, and thus easy to add to a JUCE project. Other than that, something like libserial, or even boot asio, would be a better choice, in that they are more actively maintained, and have Linux support. The library I am hosting has served my purposes, and so I keep using it. :)
